apispec/openapi-model/src/main/scala/sttp/tapir/openapi/OpenAPI.scala

```diff
@@ -114,24 +157,59 @@ case class PathItem(
       parameters = List.empty
     )
   }
+
+  def withSummary(updated: String) = copy(summary = Some(updated))
+  def withDescription(updated: String) = copy(description = Some(updated))
```

Comment:
I'm wondering if the name here should be `withDescription` or just `description`? We already have some `def description(s: String)` methods out there, and it would be good to have a uniform approach. I'm fine with either, though we would need to come up with some naming conventions.

Reply:
agreed, in http4s we have started using with* prefixed methods on our domain types.
Others might prefer the unprefixed version.

Lets choose one and stick with that.

Reply:
I think I'd go with unprefixed, since we already have them e.g. in `EndpointTransput.description` etc

Reply:
ok, should that include the add* methods as well, or do we keep them as is?

Reply:
I think `add*` is fine, as it is strictly different in what it's doing from the other ones